docs/development/extensions-core/datasketches-hll.md

```diff
@@ -59,6 +59,11 @@ druid.extensions.loadList=["druid-datasketches"]
  }
 ```
 
+The `HLLSketchBuild` aggregator builds a datasketch from the input column specified. If used during ingestion, this
+will result in Druid storing pre-generated HLL Sketch objects in the datasource, rather than the original value itself.
+If used at query time on an existing dimension, the resulting column can be used as an intermediate dimension by the
+post-aggregators below.
+
```

Comment:
```suggestion
The `HLLSketchBuild` aggregator builds a datasketch from the specified input column. When used during ingestion, Druid stores pre-generated HLL Sketch objects in the datasource instead of the original values.
When applied at query time on an existing dimension, you can use the resulting column as an intermediate dimension by the [post-aggregators](#post-aggregators).

```
Thanks for the clarification/ contribution @cloventt ! I've suggest some stylistic changes.

Reply:
I made a few minor edits to the suggestion above for consistency of language and style. One question: What does "original values" refer to? The raw data that made up the sketch?

Reply:
That was the way I understood it. It stores the sketch/aggregation.

Reply:
Style changes commited in df7d508.